src/wp-includes/script-loader.php

```diff
@@ -2541,8 +2541,21 @@ function wp_enqueue_global_styles() {
 		 * and add it before the global styles custom CSS.
 		 */
 		remove_action( 'wp_head', 'wp_custom_css_cb', 101 );
+
 		// Get the custom CSS from the Customizer and add it to the global stylesheet.
-		$custom_css  = wp_get_custom_css();
+		$custom_css = wp_get_custom_css();
+		if ( is_customize_preview() ) {
+			/*
+			 * When in the Customizer preview, wrap the Custom CSS in milestone comments to allow customize-preview.js
+			 * to locate the CSS to replace for live previewing. Make sure that the milestone comments are omitted from
+			 * the stored Custom CSS if by chance someone tried to add them, which would be highly unlikely, but it
+			 * would break live previewing.
+			 */
+			$before_milestone = '/*BEGIN_CUSTOMIZER_CUSTOM_CSS*/';
+			$after_milestone  = '/*END_CUSTOMIZER_CUSTOM_CSS*/';
+			$custom_css       = str_replace( array( $before_milestone, $after_milestone ), '', $custom_css );
+			$custom_css       = $before_milestone . $custom_css . $after_milestone;
```

Comment:
```suggestion
			$custom_css       = "\n{$before_milestone}{$custom_css}\n{$after_milestone}";
```

It's outside the diff so I can't make the suggestion, but similar for below:

```php
$stylesheet .= "\n" . wp_get_global_stylesheet( array( 'custom-css' ) );
```